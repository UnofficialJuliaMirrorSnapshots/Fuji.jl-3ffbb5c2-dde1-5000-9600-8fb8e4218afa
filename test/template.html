<html>
<head>
    <title>Hello, {% name %}!</title>
</head>
<body>
    <p>Your name is {% name %} and 2 + 2 = {% 2 + 2 %}</p>
</body>
</html>
