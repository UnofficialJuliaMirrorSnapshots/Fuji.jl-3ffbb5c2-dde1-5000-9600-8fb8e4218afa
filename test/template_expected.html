<html>
<head>
    <title>Hello, Jack!</title>
</head>
<body>
    <p>Your name is Jack and 2 + 2 = 4</p>
</body>
</html>
